www/nginx/src/opnsense/mvc/app/models/OPNsense/Nginx/Nginx.xml

```diff
@@ -698,6 +711,10 @@
           </check001>
         </Constraints>
       </default_server>
+      <ssl_reject_handshake type="BooleanField">
```

Comment:
the convention in the plugin is that the fields are called tls_* as well to have the tls also in the API calls. so please rename.
```suggestion
      <tls_reject_handshake type="BooleanField">
```